ambari: Add jdeb support

nareshgbhat opened the pull request:
This patch add jdeb (debian) package creation support for the following
packages.
-ambari-funtest
-ambari-logsearch
-ambari-metrics-grafana

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

## What changes were proposed in this pull request?

Add jdeb support patch to trunk.

## How was this patch tested?

The patch has been applied, build and tested on Debian 9 stretch.

Please review [Ambari Contributing Guide](https://cwiki.apache.org/confluence/display/AMBARI/How+to+Contribute) before opening a pull request.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2913/
Test PASSed.

nareshgbhat commented:
@adoroszlai  Can we merge the pull request ?

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/3230/
Test FAILed.
Test FAILured.

oleewere commented on ambari-logsearch/pom.xml:
```diff
@@ -131,6 +131,28 @@
+       <plugin>
+         <groupId>org.vafer</groupId>
+         <artifactId>jdeb</artifactId>
+         <version>1.0.1</version>
```
can you use version 1.4 for infra and logsearch?
note: just let you know, after 2.7.1 the ambari repo will be splitted into ambari, ambari-metrics,ambari-logsearch, ambari-infra, so those wont be built together (so possibly these changes wont be required for trunk)

nareshgbhat replied:
OK sure I will fix it and re-push the patch, Thanks for the suggestion @oleewere

nareshgbhat replied:
@oleewere my apologies for delay.  As you have suggested I have modified and given pull request only for v2.7 branch https://github.com/apache/ambari/pull/1840
Should we close this pull request  because it is for trunk ?

oleewere replied:
maybe its ok to merge this to trunk, but long term it won't matter that much. on logsearch/infra side we want to move to gradle if possible (after the repo split), jdeb/rmp plugins will be used there properly with the right tasks

nareshgbhat replied:
@oleewere ok Thanks,  I have updated the pull request.  Now the patch contains jdeb v1.4 for ambari-logsearch and ambari-infra

nareshgbhat committed:
AMBARI-23903: Add jdeb support

This patch add jdeb (debian) package creation support for the following
packages.
-ambari-funtest
-ambari-infra
-ambari-logsearch
-ambari-metrics-grafana
-ambari-metrics-host-aggregator

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/3670/
Test PASSed.
